pyigm/guis/igmguesses.py

```diff
@@ -1771,7 +1792,10 @@ def from_igmguesses_to_joebvp(infile, outfile):
     # Components
     comp_list = []
     for ii, key in enumerate(igmg_dict['cmps'].keys()):
-        comp = AbsComponent.from_dict(igmg_dict['cmps'][key], chk_sep=False, chk_data=False, chk_vel=True)
+        # QtCore.pyqtRemoveInputHook()
+        # pdb.set_trace()
+        # QtCore.pyqtRestoreInputHook()
+        comp = AbsComponent.from_dict(igmg_dict['cmps'][key], chk_sep=False, chk_data=False, chk_vel=False)
```

Comment:
Maybe explain why to have chk_vel=False here.  Unless it is obvious (it isn't to me).